### ISSUE TITLE
Merge into upstream

### ISSUE THREAD

**Issue by mjsir911** [rsn8887/OpenTTD#1]
Any plans to merge this into the main openttd repo? that would be pretty cool.

Thanks for all that you do!

**Comment by rsn8887** [comment]
Yes I tried to rebase upstream and make a PR, but they decided to remove libtimidity support in the upstream repo around March 2019, just after I forked. Libtimidity is needed for music on Switch. This causes a whole lot of merge errors.

I will try again soon.

**Comment by kevinfields777** [comment]
OpenTTD 1.11.0 just launched, might be a good time to jump back in and give it another shot. LOTS of updates and changes to make it faster.

**Comment by kevinfields777** [comment]
And FYI, OpenTTD 13 is going to be launching soon.
